backend.go

```diff
@@ -54,16 +59,105 @@ func backend() *azureAuthBackend {
 		},
 		Paths: framework.PathAppend(
 			[]*framework.Path{
-				pathLogin(b),
-				pathConfig(b),
+				pathLogin(&b),
+				pathConfig(&b),
+				pathRotateRoot(&b),
 			},
-			pathsRole(b),
+			pathsRole(&b),
 		),
+		WALRollback: b.walRollback,
```

Comment:
Azure secrets sets the [WALRollbackMinAge](https://github.com/hashicorp/vault-plugin-secrets-azure/blob/main/backend.go#L70-L72) to 10 minutes. I don't think we need it to be 10 minutes here, but I do think we should set this value. Otherwise, the rollback function could try to rollback a root credential rotation that's currently in flight.

I'm thinking 3 minutes is sufficient. That gives the periodic func at least 1 attempt to complete the rotation.

Reply:
Added a min age of 3 minutes to the wal rollback 🙏🏼